sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,244 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Get-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: itambakhe
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+
+# Get-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports about insights on sites protected and access denials by Restricted access control policy when no report ID is present and to view or download the report if report ID is present.
```

Comment:
this sentence is hard to read. Consider making it shorter -- maybe break it into two

Reply:
suggestion from Copilot (obvious rewrite it your way so its clear)

```
This cmdlet allows the administrator to check the status of all active and available reports on insights related to site protection and access denials enforced by restricted access control.
```

Reply:
Resolved